webpack.config.js

```diff
@@ -48,7 +48,7 @@ const config = {
       },
       {
         test: /(\.(js|ts)x?$)/,
-        include: /src/,
+        include: path.resolve('./src'),
```

Comment:
@mzikherman - i _think_ this might have fixed that weird path issue when yarn watching reaction